src/pineko/parser.py

```diff
@@ -35,7 +71,54 @@ def _load_yaml(yaml_file):
     return ret
 
 
-def get_yaml_information(yaml_file, grids_folder):
+def _apfelcomb_compatibility_flags(gridpaths, metadata):
+    """
+    Prepare the apfelcomb-pineappl compatibility fixes
+
+    Returns
+    -------
+        apfelcomb_norm: np.array
+            Per-point normalization factor to be applied to the grid
+            to be compatible with the data
+        apfelcomb_repetition_flag: bool
+            Whether the fktable is a single point which gets repeated up to a certain size
+            (for instance to normalize a distribution)
+        shift: list(int)
+            Shift in the data index for each grid that forms the fktable
+    """
+    operands = metadata["operands"]
+    apfelcomb_norm = None
+    if norms := metadata.get("apfelcomb_norm"):
+        # There's no easy way for an fktable to know its role in given operation:
+        for factors, grids in zip(norms, operands):
+            # Now check, in case of an operation, what is our index in this operation
+            if len(grids) == len(gridpaths) and all(
+                f.name == f"{o}.{EXT}" for f, o in zip(gridpaths, grids)
+            ):
+                apfelcomb_norm = np.array(factors)
+
+    # Check for the repetition flag, meaning we only want the first datapoint for this fktable
+    apfelcomb_repetition_flag = False
+    if metadata.get("repetition_flag"):
```

Comment:
Again, I'd test for `is not None`

Reply:
But I want (and also below) to not enter in the case of `None`, in the case of `False` and in the case of empty list. This checks all 3 cases. What would you put instead?

Reply:
```suggestion
    if metadata.get("repetition_flag", False):
```

If it's missing `get` will give you `False`, an explicit `null` in the yaml should not be supported (if the correct setting should be `false`).
For the empty list I'd say that the body should gracefully behave even in that case (if it's a meaningful empty list, corresponding to empty something else). If instead the empty list is an alias for `false`, only support `false`.

Reply:
In this way you're currently having an expected behavior e.g. for a `0.` normalization: it should be multiply by `0.`, not skipping.

Reply:
In any case, I believe that actually the meaningful value to deactivate is `null`, and not `false`, since there is no meaningful case in which it would be `true` (the usual non-empty value is a float, so empty should be `null`).

So my suggestion would be to drop support for false, and actually use:
```suggestion
    if metadata.get("repetition_flag") is not None:
```